resources/lang/de/Template.properties

```diff
@@ -335,6 +340,11 @@ orderHistoryOrderDetails = "Bestelldetails"
 orderHistoryOrderId = "Auftrags-ID"
 orderHistoryPaymentMethod = "Zahlungsart"
 orderHistoryPaymentStatus = "Zahlungsstatus"
+orderHistoryPaymentStatus_fullyPaid = "Bezahlt"
+orderHistoryPaymentStatus_overpaid = "Überzahlt"
+orderHistoryPaymentStatus_partlyPaid = "Teilweise bezahlt"
+orderHistoryPaymentStatus_prepaid = "Im Voraus bezahlt"
+orderHistoryPaymentStatus_unpaid = "Nicht bezahlt"
```

Comment:
@lks-mtzn maybe rename "unpaid" to "notPaid"?

Reply:
@cjohannsen das sind die stati die man vom server bekommt und die kann man so nicht einfach umbenennen